packages/meditrak-server/src/routes/importSurveys/Validator/ConfigValidator/ArithmeticConfigValidator.js

```diff
@@ -21,28 +21,65 @@ export class ArithmeticConfigValidator extends JsonFieldValidator {
     const defaultValuesPointToOtherQuestions = this.constructDefaultValuesPointToOtherQuestions(
       rowIndex,
     );
+    const existIfQuestionsInFormulaAreNonNumeric = this.hasContentIfQuestionsInFormulaAreNonNumeric(
+      rowIndex,
+    );
     const valueTranslationPointToOtherQuestions = this.constructValueTranslationPointsToOtherQuestions(
       rowIndex,
     );
 
     return {
       formula: [constructIsNotPresentOr(formulaPointsToOtherQuestions)],
       defaultValues: [constructIsNotPresentOr(defaultValuesPointToOtherQuestions)],
-      valueTranslation: [constructIsNotPresentOr(valueTranslationPointToOtherQuestions)],
+      valueTranslation: [
+        existIfQuestionsInFormulaAreNonNumeric,
+        constructIsNotPresentOr(valueTranslationPointToOtherQuestions),
+      ],
       answerDisplayText: [constructIsNotPresentOr(hasContent)],
     };
   }
 
+  /**
+   * When any questions used in the formula are non numeric, users need to provide 'valueTranslation'
+   * to translate the raw answers into numbers so that arithmetic calculation can be done.
+   * @param {*} rowIndex
+   */
+  hasContentIfQuestionsInFormulaAreNonNumeric(rowIndex) {
```

Comment:
I've fixed the bug that Sepi found (https://github.com/beyondessential/tupaia-backlog/issues/982#issuecomment-731864359). If the formula in Arithmetic question contains any questions that are non numeric, users must provide `valueTranslation` to convert them into numeric so that the arithmetic calculation can be run.

Reply:
Clever fix!